recipes/scikit-mdr/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "scikit-MDR" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "fc4e6710c7ac7a63c55641773388e43cc75bbe19f8d946bb8ac09eadb7d9eb2e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - matplotlib
+    - numpy
+    - python
+    - scikit-learn
+    - scipy
+
+test:
+  imports:
+    - mdr
+
+about:
+  home: https://github.com/EpistasisLab/scikit-mdr
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: |
+    A sklearn-compatible Python implementation of Multifactor Dimensionality
+    Reduction (MDR) for feature construction.
+
+  description: |
+    MDR is an effective feature construction algorithm that is capable of
+    modeling higher-order interactions and capturing complex patterns in data
+    sets.
+
+    MDR currently only works with categorical features and supports both binary
+    classification and regression problems. We are working on expanding the
+    algorithm to cover more problem types and provide more convenience features.
+  dev_url: https://github.com/EpistasisLab/scikit-mdr
+
+extra:
+  recipe-maintainers:
+    - bollwyvl
+    - lacava
+    - pschmitt52
```

Comment:
@pschmitt52, can you comment here that you're okay being added as [a co-maintainer](https://conda-forge.org/docs/recipe.html#maintainer-role) to skrebate and scikit-mdr?